xgmutil/stream.py

```diff
@@ -29,9 +36,7 @@ def generate(self,**kwargs):
 
         seqIDs = jnp.arange(start_seqID, end_seqID+1)
 
-        seedkey = rnd.PRNGKey(self.seed)
```

Comment:
here we should change the behavior depending on the type of `self.seed`, see my comment above.

Reply:
ok, but we need to set a precedence in case both are provided.

Reply:
looks done, except `self.seed` --> `self.seedkey`, resolving...